pkg/acceptance/terraform/azure/variables.tf

```diff
@@ -0,0 +1,115 @@
+# Number of CockroachDB instances. This is overridden by terrafarm.
+variable "num_instances" {
+  default = "0"
+}
+
+# Port used for the load balancer and backends.
+variable "sql_port" {
+  default = "26257"
+}
+
+variable "http_port" {
+  default = "8080"
+}
+
+# List of stores each with its own --store flag. (--store flags from the cockroach binary).
+variable "stores" {
+  default = "--store=data"
+}
+
+# Azure location to use.
+variable "azure_location" {
+  default = "eastus"
```

Comment:
Let's try `eastus` first to reduce potential for SSH latency issues. This is simple enough to change if it becomes an issue.